src/layout/layout.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './layout.css';
+import { UsersBar } from '../users_bar/users_bar';
+// import { get } from '../queries/fetcher';
+import MockUsers from '../mock/users.json';
+import { UserType } from '../user/user';
+
+type Props = {};
+type State = {
+  users: Array<UserType>;
+};
+
+export class Layout extends React.Component<Props, State> {
+  state = {
+    users: []
+  };
+
+  componentDidMount(): void {
+    // get('localhost/public/mockapi/users.js');
```

Comment:
Please, delete comment. We have history from git.

Reply:
fixed